.travis.yml

```diff
@@ -7,7 +7,7 @@ rvm:
   - 1.9
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.2
```

Comment:
Also see https://travis-ci.org/rails/jbuilder/jobs/83196945#L90, the latest CI test of `jbuilder` `master` branch where the RVM ruby version was specified as `2.2`.